.github/workflows/lineendings.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Check CAP Line Endings'
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  lineendings:
+    runs-on: ubuntu-latest
+    steps:
+    - run: sudo apt-get update && sudo apt-get install dos2unix
+    - uses: actions/checkout@v2
+    - run: (ls core/cap-*.md | xargs -I '{}' sh -c 'cat {} | dos2unix | cmp - {}') || (echo 'Documents listed above contain CRLF line endings, and should be LF.'; exit 1)
```

Comment:
Do we also need to enforce encoding or crlf is enough?

I see that dos2unix allows to enforce utf8bom for example.

Reply:
I'd like to enforce both, but I'd like to start with only line endings in this PR and follow up with utf8 in a separate pr once I understand the state of that better I'm the repo.

Reply:
I looked more at this, I don't think we need to enforce utf8bom, just utf8 is sufficient whether the bom is present or not. Many of our CAPs only used ASCII characters and so the distinction is not particularly useful. If any file strays outside utf8 the dos2unix tool should error on that I think.

Unless you think we should require utf8bom, or need to do more testing here, I think we're okay.